models/authDao.js

```diff
@@ -0,0 +1,31 @@
+const { appDataSource } = require("./dataSource");
+const error = require("../middlewares/errorCreater");
+
+const createUser = async (nickname, kakaoId, profileImgUrl) => {
+  
+  const user = await appDataSource.query(
+    `INSERT INTO user (
+      nickname,
+      kakao_id,
+      profile_img_url
+    ) VALUES (?,?,?)`,
+    [nickname, kakaoId, profileImgUrl]
+  );
+  
+  return user;
+};
+
+const getUserById = async (kakaoId) => {
```

Comment:
```suggestion
const getUserByKakaoId = async (kakaoId) => {
```
id가 아니라 kakao_id로 필터링중이니 명확히 적어주세요.